.changelog/3307.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+plugin/k8s: Add CPU and memory resource limits to on-demand runners through runner profiles and at install time
```

Comment:
Let's specify that these are the same units as what the K8s API expects so there's no confusion.

Reply:
Thanks for the feedback!

Addressed in `a474021`